Cargo.lock

```diff
@@ -2397,7 +2175,6 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "27abb6e4638dcecc65a92b50d7f1d87dd6dea987ba71db987b6bf881f4877e9d"
 dependencies = [
  "num-traits",
- "serde",
```

Comment:
I think this has caused some build errors. not sure how serde got removed but assume it has to do with wsts being deleted.